lib/hyrax/configuration.rb

```diff
@@ -707,7 +707,7 @@ def collection_model
     ##
     # @return [Class] the configured collection model class
     def collection_class
-      collection_model.constantize
+      collection_model.safe_constantize
```

Comment:
Switch to `safe_constantize` to catch the case where an invalid model is set in `collection_model`.